src/main/java/seedu/address/logic/commands/ListProcCommand.java

```diff
@@ -0,0 +1,61 @@
+package seedu.address.logic.commands;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.client.Client;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.*;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
+
+public class ListProcCommand extends Command {
+
+    public static final String COMMAND_WORD = "listProc";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Lists the procedures of a client in the index. "
```

Comment:
I was imagining how this would look like. Would it be better to add a newline at the end of this to allow for better user-readability?

```
": Lists the procedures of a client in the index. \n"
```